go.mod

```diff
@@ -1,23 +1,24 @@
 module github.com/brave-experiments/nitriding
 
-go 1.17
+go 1.18
 
 require (
 	github.com/brave-experiments/viproxy v0.1.0
 	github.com/go-chi/chi/v5 v5.0.7
+	github.com/hf/nitrite v0.0.0-20211104000856-f9e0dcc73703
 	github.com/hf/nsm v0.0.0-20211106132757-1ae65a6a69ae
 	github.com/mdlayher/vsock v1.1.1
 	github.com/milosgajdos/tenus v0.0.3
-	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
-	golang.org/x/sys v0.0.0-20220319134239-a9b59b0215f8
+	golang.org/x/crypto v0.0.0-20220622213112-05595931fe9d
+	golang.org/x/sys v0.0.0-20220715151400-c0bba94af5f8
```

Comment:
If these version bumps aren't essential to the code changes, it would be nicer for review if they were in a separate "Update dependencies" commit.

Reply:
Right, that's my bad. Is it ok with you to leave this as it is, to prevent a force push?

Reply:
Yes, it's fine if you want to leave it in the same commit.